impeller/tools/malioc.json

```diff
@@ -13266,6 +13263,131 @@
       }
     }
   },
+  "flutter/impeller/renderer/path_polyline.comp.vkspv": {
```

Comment:
I have no idea why the changes above are happening. I tried commenting out chagnes in compiler.cc and they still were there.